README.md

```diff
@@ -6,19 +6,19 @@ The Dataset-JSON v1.1 release can be downloaded as a [zip](https://github.com/cd
 
 ## Dataset-JSON Specification
 
-The [doc](https://github.com/cdisc-org/DataExchange-DatasetJson/tree/publish_v11/doc) folder contains the specifications of:
+The [doc](https://github.com/cdisc-org/DataExchange-DatasetJson/tree/master/doc) folder contains the specifications of:
 
-- [Dataset-JSON v1.1](doc/dataset-json1-1.html).
-- The [NDJSON](doc/dataset-json-ndjson1-1.html) representation of Dataset-JSON v1.1
-- [Compressed Dataset-JSON v1.1](doc/compressed-dataset-json-1-1.html)
+- [Dataset-JSON v1.1](https://github.com/cdisc-org/DataExchange-DatasetJson/blob/master/doc/dataset-json1-1.md).
```

Comment:
I think the previous link format was correct, i.e., **doc/dataset-json1-1.md** instead of **https://github.com/cdisc-org/DataExchange-DatasetJson/blob/master/doc/dataset-json1-1.md**